lib/rules/declaration-block-no-duplicate-properties/index.js

```diff
@@ -76,6 +77,10 @@ const isEqualValueSyntaxes = (value1, value2) => {
 		return true;
 	}
 
+	if (!(isStandardSyntaxValue(value1) && isStandardSyntaxValue(value2))) {
+		return false;
+	}
```

Comment:
[suggest] Can you add a test case to cover the new code? For example,

```scss
p { width: $a; width: calc(1 + $a); }
```

Reply:
For the current case, I don't know how to write `reject` code in test case.

Reply:
Sure. For example:

```diff
--- a/lib/rules/declaration-block-no-duplicate-properties/__tests__/index.js
+++ b/lib/rules/declaration-block-no-duplicate-properties/__tests__/index.js
@@ -307,6 +307,7 @@ testRule({
 		{ code: 'p { width: min(10px, 11px); width: max(10px, 11px); }' },
 		{ code: 'p { width: calc((10px + 2px)); width: calc((10rem + 2rem)); }' },
 		{ code: 'p { width: calc((10px + 2px) + 10px); width: calc((10rem + 2rem) + 10px); }' },
+		{ code: 'p { width: $a; width: calc(1 + $a); }', description: 'using SCSS variables' },
 	],
 
 	reject: [
```

Reply:
Done, if it no need to configure `reject`.

Reply:
Thanks!